property/example_test.go

```diff
@@ -79,3 +83,65 @@ func ExampleCollector_Retrieve() {
 	})
 	// Output: host has 2 vms: DC0_H0_VM0 DC0_H0_VM1
 }
+
+func ExampleWait() {
+	simulator.Run(func(ctx context.Context, c *vim25.Client) error {
+		pc := property.DefaultCollector(c)
+		defer func() {
+			_ = pc.Destroy(ctx)
```

Comment:
Also no need to Destroy the `DefaultCollector` singleton, I think it'll actually fail against real vCenter or is a noop at least.

Reply:
oh, so destroying pc would only be recommended when having a dedicated pc (ie not default)? good to know!